src/script/interpreter.cpp

```diff
@@ -2114,6 +2200,15 @@ bool VerifyScript(const CScript &scriptSig, const CScript &scriptPubKey,
             return false;
```

Comment:
How does the above `EvalScript` work when there's no valid script in the scripstig?